php_bson.h

```diff
@@ -60,11 +69,7 @@ typedef struct {
 	bool                     is_visiting_array;
 } php_phongo_bson_state;
 
-#if PHP_VERSION_ID >= 70000
-#define PHONGO_BSON_STATE_INITIALIZER  { {{ 0 }}, { PHONGO_TYPEMAP_NONE, NULL, PHONGO_TYPEMAP_NONE, NULL, PHONGO_TYPEMAP_NONE, NULL}, NULL, 0 }
-#else
-#define PHONGO_BSON_STATE_INITIALIZER  { NULL, { PHONGO_TYPEMAP_NONE, NULL, PHONGO_TYPEMAP_NONE, NULL, PHONGO_TYPEMAP_NONE, NULL}, NULL, 0 }
-#endif
+#define PHONGO_BSON_STATE_INITIALIZER  { 0 }
```

Comment:
Do you recall what compilers were affected (e.g. clang, MS)?

I can always revert this, but I'd be curious to test it.